ubuntudesign/www.ubuntu.com
# use juju cards + lint cloud.css

## Done

* have /cloud/juju using the embeddable card
* linted and cleaned up cloud.css a lot - mostly removing un-used css and in-lining media queries
* fixed some broken class names and css on the training page for the hero image
* removed some share partials that are no longer used

## QA

1. go to /cloud/juju and see that it is using the juju cards and look good
2. go to /cloud/training and see that the medals look better
3. go through the rest of the cloud section and see that it's the same as it was

## Issue / Card

Fixed #829
